core/src/main/java/io/questdb/cairo/ImplicitCastException.java

```diff
@@ -0,0 +1,130 @@
+/*******************************************************************************
+ *     ___                  _   ____  ____
+ *    / _ \ _   _  ___  ___| |_|  _ \| __ )
+ *   | | | | | | |/ _ \/ __| __| | | |  _ \
+ *   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
+ *    \__\_\\__,_|\___||___/\__|____/|____/
+ *
+ *  Copyright (c) 2014-2019 Appsicle
+ *  Copyright (c) 2019-2022 QuestDB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ ******************************************************************************/
+
+package io.questdb.cairo;
+
+import io.questdb.std.Chars;
+import io.questdb.std.FlyweightMessageContainer;
+import io.questdb.std.ThreadLocal;
+import io.questdb.std.str.StringSink;
+
+public class ImplicitCastException extends RuntimeException implements FlyweightMessageContainer {
+    private static final ThreadLocal<ImplicitCastException> tlException = new ThreadLocal<>(ImplicitCastException::new);
+    private static final StackTraceElement[] EMPTY_STACK_TRACE = {};
+    protected final StringSink message = new StringSink();
+
+    public static ImplicitCastException inconvertibleValue(int columnNumber, double value, int fromType, int toType) {
```

Comment:
unused method

Reply:
sorry, my bad, I had started reviewing an older version and I only moved to the very latest after I made the comment.